src/PageObject/Page.php

```diff
@@ -213,8 +213,11 @@ protected function verifyResponse()
      */
     protected function verifyUrl(array $urlParameters = array())
     {
-        if ($this->getDriver()->getCurrentUrl() !== $this->getUrl($urlParameters)) {
-            throw new UnexpectedPageException(sprintf('Expected to be on "%s" but found "%s" instead', $this->getUrl($urlParameters), $this->getDriver()->getCurrentUrl()));
+        $currentUrl = $this->getDriver()->getCurrentUrl();
+        $expectedUrl = $this->getUrl($urlParameters);
```

Comment:
Can we swap the order of these two definitions? I know the time to call `getUrl` is neglectable, but why not?

Reply:
Done - I squashed and pushed again. The expected URL is some fixed value anyway, so actually it should not matter when that is retrieved.